packages/template-retail-react-app/app/components/_app-config/index.jsx

```diff
@@ -68,7 +68,7 @@ const AppConfig = ({children, locals = {}}) => {
             headers={headers}
             // Uncomment 'enablePWAKitPrivateClient' to use SLAS private client login flows.
             // Make sure to also enable useSLASPrivateClient in ssr.js when enabling this setting.
-            // enablePWAKitPrivateClient={true}
+            enablePWAKitPrivateClient={true}
```

Comment:
probably want to revert this before we merge but leaving it now for testing purposes